src/components/ae-input-plain/ae-input-plain.vue

```diff
@@ -3,13 +3,16 @@
     :class="['ae-input-plain', fill]"
     :placeholder="placeholder"
     :value="value"
-    @input="$emit('input', $event.target.value)"
+    @input="propagateValue"
```

Comment:
I don't think that we should extract `$emit('input', $event.target.value)`, looks like that `$event` is defined specially for this case.

Reply:
At least would be better to make `propagateValue` a directive instead of the method of `events` mixin.

Reply:
@davidyuk maybe we can do this later! Its just a workaround to make `v-model` work